csrc/scheduler/resize.cpp

```diff
@@ -0,0 +1,135 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+
+#include <debug.h>
+#include <instrumentation.h>
+#include <ir/utils.h>
+#include <scheduler/cache_policy_refiner.h>
+#include <scheduler/debug_utils.h>
+#include <scheduler/mark_aliases.h>
+#include <scheduler/pointwise_utils.h>
+#include <scheduler/registry_utils.h>
+#include <scheduler/resize.h>
+#include <scheduler/runtime_info.h>
+#include <scheduler/tools/inlining.h>
+#include <scheduler/tools/loop_domain_scheduler.h>
+#include <scheduler/tools/resize_utils.h>
+#include <val_graph_visitor.h>
+
+namespace nvfuser {
+
+bool ResizeScheduler::canScheduleCompileTime(Fusion* fusion) {
```

Comment:
In this initial version, I'm trying to make it very restrictive. Will have several follow-up PRs to schedule the whole RoPE module.